### PR TITLE
Attribute credit for CIS content

### PR DESCRIPTION
**Pull request by yuumasato**
#### Description:

- Attribute credit to Center for Internet Security for Benchmark contents
- Update the description text a bit.

This gets correctly rendered in OAA, and in HTML Guides.
trademarks

guide_trademarks

### CODE REVIEW

File: rhel7/profiles/cis.profile [hunk 1]
```diff
@@ -3,9 +3,11 @@ documentation_complete: true
 title: 'CIS Red Hat Enterprise Linux 7 Benchmark'
 
 description: |-
-    This baseline aligns to the Center for Internet Security
-    Red Hat Enterprise Linux 7 Benchmark, v2.2.0, released
-    12-27-2017.
+    This profile defines a baseline that aligns to the Center for Internet Security®
+    Red Hat Enterprise Linux 7 Benchmark, v2.2.0, released 12-27-2017.
```
Comment (redhatrises) [inline rhel7/profiles/cis.profile:7]: Is this date and benchmark version accurate?

Comment (yuumasato) [reply rhel7/profiles/cis.profile:7]: They are correct.
But I just found out there is a 2.2.0.1, from Oct 29th 2018, :/

**Comment by redhatrises** [comment]
Merging as is. Thanks.

File: rhel7/profiles/cis.profile (hunk 1 above)
Comment (yuumasato) [reply rhel7/profiles/cis.profile:7]: v2.2.0.1 seems to be unavailable for download.
And support discussion mentions there is virtually no difference between 2.2.0 and 2.2.0.1

Comment (ggbecker) [inline rhel7/profiles/cis.profile:7]: small nitpick `Benchmark` here doesn't have the ™ (trademark symbol as there  is for rhel8 profile)

**Pull request merged**